lib/index.js

```diff
@@ -273,6 +281,49 @@ exports.createStorage = async (ctx) => {
     }
 };
 
+function generateStorageUrl(functionCtx) {
+    return `${functionCtx.configuration.fusebit_storage_audience}/v1/account/${functionCtx.fusebit_storage_account_id}/subscription/${functionCtx.configuration.fusebit_storage_subscription_id}/storage/${functionCtx.configuration.fusebit_storage_id}`;
+}
+
+exports.deleteStorage = async (functionCtx) => {
+    debug('DELETE STORAGE');
+
+    const issuerId = functionCtx.fusebit_storage_issuer_id;
+    const subject = functionCtx.fusebit_storage_subject;
+    const accountId = functionCtx.fusebit_storage_account_id;
```

Comment:
Not sure what `functionCtx` represents at this point, but should this not be `functionCtx.configuration.{...}`?